examples/js/loaders/TDSLoader.js

```diff
@@ -43,13 +43,20 @@ THREE.TDSLoader.prototype = {
 
 		var scope = this;
 
-		var path = this.path;
-		if ( this.path === "" )	{
+
+		var path;
+		if ( this.path === undefined ) {
 
 			path = THREE.Loader.prototype.extractUrlBase( url );
+			url = url.replace( path, '' );
+
+		} else {
+
+			path = this.path;

```

Comment:
It should be now possible to write it like this (`GLTFLoader` style):
```javascript
var path = this.path !== undefined ? this.path : THREE.Loader.prototype.extractUrlBase( url );
```

Reply:
No we can't, see https://github.com/mrdoob/three.js/issues/12730 
When I write it like ```GLTLoader``` style we set the path a few lines below so we get a "double" path, so that is why I would suggest line 51 ```url = url.replace( path, '' );``` to set the url to the file name after extracting the path and setting it at line 64

Reply:
And when you just remove `loader.setPath( path ); ` in `TDSLoader`?

Reply:
Already thought about it, may be a good idea, but have to test it first